### ISSUE TITLE
Dependency issues with Windows.AI.MachineLearning.pdb

### ISSUE THREAD

**Issue by extremilio** [microsoft/onnxruntime#3712]
Hi,

We are having issues with .NET core 2.2 web projects referencing Microsoft.ML.OnnxRuntime (1.2.0).
The web projects is hosted in Azure as an app service (Microsoft.Web/sites).

The nuget package has a dependency for Windows.AI.MachineLearning.pdb, and since we remove all .pdbs before we publish to cloud, the web app crashses with a 502.5 error code.

Solutions we've found:
- Leaving the _\bin\runtimes\win-x86\Windows.AI.MachineLearning.pdb_ (obviously)
- Removing the _\bin\runtimes\win-x86\Windows.AI.MachineLearning.pdb_ reference from the *deps.json file

We'd rather not keep a .pdb that is over 50 Mb per runtime project referencing the nuget, and we haven't found any other solution to the *deps.json reference than removing it post build with Powershell.

_We also see some very large files in \bin\runtimes\ for osx and linux, which we of course can remove post build, but it feels a bit odd that the runtimes directory is over 600 Mb for some dependencies._ Edit: realized this is most certainly on us for not publishing to a specific runtime/framework in TeamCity.

Also tested around with [ExcludeAssets settings](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files) in the *.csproj / nuget reference without any luck.

This is a related issue / solution we found:
https://github.com/klalle/Microsoft.ML.onnxruntime_UWP_fix

So basically we _can_ solve the issue, but would like to know the reason for the .pdb dependency, and what your recommendations / solution to the above is? It's not an issue we've ever had with other Nuget packages.

Kind regards,
Emil

**Comment by pranavsharma** [comment]
@smk2007 can you please take a look?

**Comment by martinb35** [comment]
@extremilio - can you try again with the latest NuGet package? We have removed the pdbs.
